code/modules/antagonists/cult/runes.dm

```diff
@@ -315,7 +310,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 				to_chat(M, span_cultlarge("\"I accept this sacrifice.\""))
 			else
 				to_chat(M, span_cultlarge("\"I accept this meager sacrifice.\""))
-	
+
```

Comment:
Remove these changes, these have nothing to do with the PR and just seem like useless revertions and the introduction of a massive exploit for cults.

Reply:
I may have been stupid and committed from an older repo